### DISCUSSION TITLE
is possible disabled the metrics powertools through a environment variable in cloudformation?

### DISCUSSION THREAD

**Discussion by carlosLMZarego** [aws-powertools/powertools-lambda-python#2986] [Q&A]
Hi...I'm using metrics,tracer and logger in my cloudformation, I found that tracer and logs can be disabled passing some parameters in my CF yaml template...but I didn't found any information about how can I disable the metrics, is it possible? thanks

**Comment by rubenfonseca** [comment]
Hi @carlosLMZarego thank you for opening this message! As far as I know there's currently no mechanism to disable metrics through an environment variable. Can you share a little bit more of your use case? Maybe we can consider adding the feature if it makes sense.

**Comment by leandrodamascena** [comment]
Hello @rubenfonseca and @carlosLMZarego. I'm converting this discussion into an issue because it makes sense for us to create an environment variable to disable creating custom metrics.

Just a short explanation: We will disable the ability to create custom metrics using our `Metrics` or `DatadogMetrics` provider, but we cannot disable default metrics created by AWS services.

**Comment by leandrodamascena** [comment]
Closing this discussion.

More information: https://github.com/aws-powertools/powertools-lambda-python/issues/2996
